src/System.Text.RegularExpressions/tests/CaptureCollectionTests.netcoreapp.cs

```diff
@@ -164,6 +164,7 @@ public static void IList_IsReadOnly()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot)]
```

Comment:
Would it be clearer to have a ConditionalFact dependent on something like SupportsCompiledRegex property?

Reply:
Or PlatformDetection.IsRefEmitSupported maybe

Reply:
Actually I'm a little unclear why this is UapAot not Uap, if there's no compilation support for either of them?

Reply:
This class doesn't need compiled regex for the tests but it was disabled before on uap and netfx so I reenabled them. I mentioned that in my initial post as "Reenable more tests for uap/uapaot". The tests succeed on uap but fail on uapaot.